packages/@glimmer/runtime/lib/vm/element-builder.ts

```diff
@@ -83,6 +84,7 @@ export class NewElementBuilder implements ElementBuilder {
   [CURSOR_STACK] = new Stack<Cursor>();
   private modifierStack = new Stack<Nullable<ModifierInstance[]>>();
   private blockStack = new Stack<LiveBlock>();
+  private remoteStateStack: Object[];
```

Comment:
Shouldn't be necessary (and even if it were, `Object` would definitely be the wrong type)

Reply:
Still relevant (maybe re-read the comment about using the bounds object to uniquely identify the node?)

Reply:
removed